src/_scss/pages/state/_contentWrap.scss

```diff
@@ -1,33 +1,198 @@
-            font-weight: 300;
-            line-height: rem(47);
-        }
-
-        .state-section__description {
-            font-size: $lead-font-size;
-            font-weight: 300;
-            line-height: rem(25);
-            padding-bottom: rem(35);
-        }
-
-        hr.results-divider {
+	background-color: $color-white;
+	color: $color-base;
+
+	.state-section {
+		h3.state-section__title {
+			margin: 0;
+			font-size: $h3-font-size;
+			font-weight: 300;
+			line-height: rem(47);
+		}
+
+		.state-section__description {
+			font-size: $lead-font-size;
+			font-weight: 300;
+			line-height: rem(25);
+			padding-bottom: rem(35);
+		}
+
+		.state__controls-desktop {
+			margin-bottom: $global-margin * 2;
+
+			@import '../../components/roundedtoggle';
+
+			@media(min-width: $large-screen) {
+				display: flex;
+				justify-content: end;
+			}
+
+			.state__line-div {
+				@media(min-width: $large-screen) {
+					border-right: 1px solid $color-gray-lighter;
+					line-height: 24px;
+					height: 24px;
+					margin-left: 8px;
+				}
+			}
+
+			// overrides to level out the text in the row
+			.rounded-toggle__wrapper {
+				margin-top: rem(2);
+			}
+
+			.accordion-container {
+				.accordion {
+					.heading {
+						padding-top: rem(10);
+					}
+				}
+			}
+
+			.accordion-container {
+				@media(max-width: $tablet-screen) {
+					margin-bottom: rem(4);
+				}
+				@media(min-width: $tablet-screen) {
+					margin-left: rem(10);
+				}
+			}
+
+			.accordion-container .heading {
+				height: 14px;
+				color: $color-primary;
+				border: none;
+				font-size: $font-size-14;
+				max-width: rem(120);
+				padding: 0;
+				margin: 0;
+				justify-content: unset !important;
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+
+				@media(min-width: $tablet-screen) {
+					padding-top: rem(8);
+				}
+			}
+
+			.accordion-container .plus {
+				width: 10px;
+				height: 10px;
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+			}
+
+			.accordion-container .minus {
+				width: 10px;
+				height: 10px;
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+			}
+
+			.accordion-container .toggle {
+				color: $color-primary;
+				font-size: $font-size-14;
+				font-weight: $font-semibold;
+				padding-left: rem(6);
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+
+				@media(min-width: $tablet-screen) {
+					padding-right: rem(6);
+				}
+			}
+
+			.accordion-container .open .content {
+				display: none;
+			}
+		}
+
+		.state__what-content {
+			border-left: 8px solid $color-primary-alt-light;
+			border-top: 1px solid $color-primary-alt-light;
+			border-right: 1px solid $color-primary-alt-light;
+			border-bottom: 1px solid $color-primary-alt-light;
+			background-color: #e0f3f8;
```

Comment:
is there a uswds variable for this hex value?

Reply:
i couldn't find one here https://designsystem.digital.gov/design-tokens/color/system-tokens